addons/supplies/functions/fnc_loadTraits.sqf

```diff
@@ -0,0 +1,33 @@
+#include "script_component.hpp"
+/*
+ * Author: JoramD
+ * Load traits based on trainings on the website.
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call tac_supplies_fnc_loadTraits
+ *
+ * Public: No
+ */
+
+params ["_player"];
+
+// TODO: Add check if appollo is enabled
+if EGVAR(apollo,enabled) then {
```

Comment:
```suggestion
if (!EGVAR(apollo,enabled)) exitWith {};

// the rest of the code
```

Reply:
This component should require `tac_apollo` as well. However, even better if we use `ACEFUNC(common,isModLoaded)` before checking the enabled variable and using a function from that component.